sql/core/src/main/scala/org/apache/spark/sql/streaming/ui/StreamingQueryStatisticsPage.scala

```diff
@@ -264,7 +265,12 @@ private[ui] class StreamingQueryStatisticsPage(parent: StreamingQueryTab)
           </tr>
         </tbody>
       </table>
-    // scalastyle:on
+    } else {
+      <div id="empty-streaming-query-message">
+        <b>No visualization information available because there is no batch.</b>
```

Comment:
`there is no batch` - this might be slightly misleading if the query was running with some batches and stopped, and restarted with checkpoint. I can't think of good alternative though as of now...

Reply:
So how about just simply saying "No visualization information available" ?

Reply:
I'm OK with that - maybe hearing more voices gives more options, hopefully.